portfolio/src/main/webapp/script.js

```diff
@@ -12,13 +12,84 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(init);
+
+    function init() {
+        //raw data
+        var languages = [['Language', 'Proficiency Level', { role: 'style' }, { role: 'annotation' } ],
```

Comment:
similar to how you did for greetings, I think its better to have these defined outside the method since they are not dynamic

Reply:
fixed!